docs/caph/01-getting-started/02-quickstart/01-prerequisites.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: Prerequisites
+metatitle: Understanding the Cluster API for Hetzner Cluster Creation
+sidebar: Prerequisites
+description: Simplify provisioning, upgrading, and operating Kubernetes clusters on Hetzner infrastructure with Cluster API for a production-ready setup.
```

Comment:
We don't need to use this kind of marketing speech here. I wouldn't say that CAPH simplifies upgrading, much less in a production scenario.